components/wifi_connector/wifi_ap.cpp

```diff
@@ -0,0 +1,72 @@
+#include "wifi_ap.hpp"
+#include "event_handlers.hpp"
+
+WifiAP::WifiAP()
+{
+  cfg_ap_ = {};
+  cfg_ap_.ap = {};
+  cfg_ap_.ap.ssid_hidden = 0;
+  cfg_ap_.ap.authmode = WIFI_AUTH_WPA_WPA2_PSK;
+
+  this->max_connections(4);
+
+  EventHandlers::add(WIFI_EVENT, this);
+}
+
+void WifiAP::ssid(std::string ssid)
+{
+  if (ssid.length() < 32) {
+    std::copy(ssid.begin(), ssid.end(), cfg_ap_.ap.ssid);
+    ssid_ = ssid;
+  }
+  else
+  {
+    ssid_ = "";
```

Comment:
Is an empty string a correct fallback if the given string is longer than 32 chars? Would a neater solution be to create an AP with the allowed length of the provided SSID?